assets/component-image-with-text.css

```diff
@@ -51,8 +51,6 @@
 .image-with-text__media--placeholder:after {
   content: '';
   position: absolute;
-  width: 100%;
-  height: 100%;
```

Comment:
This was not doing anything, that's why we're removing it ?

Reply:
Removing it fixes a small bug with the placeholder shadow that was preexisting and unrelated to this PR, but it's much uglier when removing persistent gradient on the media like we're doing in this PR https://screenshot.click/19-59-lvmkd-swxu2.mp4